src/c++/perf_analyzer/genai-pa/genai_pa/llm_metrics.py

```diff
@@ -137,8 +139,30 @@ def _use_exact_length(self, value: str):
         return formatted_value[:exact_value_length]
 
     def pretty_print(self):
+        table = Table(title="PA LLM Metrics")
+
+        table.add_column("Statistic", justify="right", style="cyan", no_wrap=True)
+        stats = ["avg", "min", "max", "p99", "p95", "p90", "p75", "p50", "p25"]
+        for stat in stats:
+            table.add_column(stat, justify="right", style="green")
+
+        metrics = ["inter_token_latency", "time_to_first_token"]
+        for metric in metrics:
+            formatted_metric = metric.replace("_", " ").capitalize()
+            is_time_field = self._is_time_field(metric)
+            if is_time_field:
+                formatted_metric += " (ns)"
+            row_values = [formatted_metric]
+
+            for stat in stats:
+                value = self.__dict__.get(f"{stat}_{metric}", -1)
+                row_values.append("{:,.0f}".format(value))
+            table.add_row(*row_values)
+
+        console = Console()
+        console.print(table)
+
         field_stats = {}
```

Comment:
is field_stats used?

Reply:
Thanks for catching this! I missed some of the old implementation code. It should be cleaner now. :)